pkg/cli/common.go

```diff
@@ -54,6 +54,7 @@ type BudResults struct {
 	DisableContentTrust bool
 	File                []string
 	Format              string
+	HttpProxy    bool
```

Comment:
You need to run your code changes through `gofmt -w -s`, to be able to pass tests.

Reply:
Fixed